HealthMonitoring/Security/CredentialsProvider.cs

```diff
@@ -1,28 +1,37 @@
 ﻿using System;
+using System.Collections.Specialized;
 using System.Configuration;
 
 namespace HealthMonitoring.Security
 {
     public class CredentialsProvider : ICredentialsProvider
     {
-        private string _adminTokenConfigKey = "AdminMonitorPrivateToken";
-        private string _adminIdConfigKey = "AdminMonitorId";
-        private string _pullMonitorTokenConfigKey = "PullMonitorPrivateToken";
-        private string _pullMonitorIdConfigKey = "PullMonitorId";
+        private const string _adminTokenConfigKey = "AdminMonitorPrivateToken";
```

Comment:
So the SecurityRole enum has been renamed but the credentials provider is still using the old names. Could you please sync it up?